src/francesinha.md

```diff
@@ -0,0 +1,45 @@
+# Francesinha
+
+Francesinha is a traditional sandwich from the city of Porto. The francesinha in its special sandwich variation consists of sausage, fresh sausage, ham, cold meat and beef steak, covered with melted cheese.
+
+## Based Ingredients
+
+- bread
+- linguiça
```

Comment:
```suggestion
- linguiça (type of Portuguese sausage)
```

Maybe it's just me, but I had no idea what `linguiça` was, so having just a short description might be helpful to others.